cpp/modmesh/pilot/R3DWidget.cpp

```diff
@@ -113,12 +113,6 @@ void R3DWidget::updateWorld(std::shared_ptr<WorldFp64> const & world)
     cameraController()->setFarPlane(dist + half_height * 2);
 }
 
-void R3DWidget::closeAndDestroy()
```

Comment:
It is hard to make sense to call `deleteLater()` from within.  Remove the helper for now.